src/home/room_screen.rs

```diff
@@ -1004,6 +1029,28 @@ impl Widget for RoomScreen {
                         }
                     }
                 }
+
+                if let Some(timeline_update) = action.downcast_ref::<TimelineUpdate>() {
+                    match timeline_update {
+                        TimelineUpdate::NewItems {items, ..} => {
+                            let portal_list = self.portal_list(id!(timeline.list));
+                            if !portal_list.is_at_end() {
+                                self.unread_message_count += items.len();
+                                self.view(id!(jump_to_bottom_view)).set_visible(true);
+                                let label_text  = if self.unread_message_count > 0 {
+                                    format!("{} new messagers", self.unread_message_count)
```

Comment:
typo. Also should probably handle the case of "1 new message" vs "N new messages" where N > 1.
```suggestion
                                    format!("{} new messages", self.unread_message_count)
```

Reply:
Typo fixed, will look into the cases you mentioned